students/M_Sunday/lesson03/slicing_lab_exercise.py

```diff
@@ -0,0 +1,59 @@
+# lesson 03 - slicing lab exercise
+
+seq1 = "what a beautiful day"
+seq2 = ['red', 'blue', 'orange', 'green', 'yellow', 'cyan', 'magenta', 'violet', 'aqua']
+seq3 = (1, 2, 3, 4, 5, 6, 7, 8, 9)
+
+
+def last_first(seq):
+    # This function swaps the first and last items in a sequence
+    return seq[-1:] + seq[1:-1] + seq[:1]
+
+
+def every_other(seq):
+    # This function returns every other item in a sequence
+    return seq[::2]
+
+
+def every_other_short(seq):
+    # This function returns every other item in a sequence after
+    # removing the first four and last four items
+    return seq[4:-4:2]
+
+
+def reversed1(seq):
+    # This function reverses the item order in a sequence
+    return seq[::-1]
+
+
+def third_swap(seq):
+    # This function orders the items in the sequence with the middle
+    # third of the items, the last third, and then the first third
+    stop = int(len(seq)/3)
```

Comment:
You can use the floor division operator `//` here to dealing with numbers that aren't divisible by three